optuna_dashboard/ts/graphUtil.ts

```diff
@@ -45,7 +45,7 @@ const getAxisInfoForCategoricalParams = (
   )
 
   const indices = distribution.choices
-    .map((c) => c?.toString() ?? "null")
+    .map((c) => c?.value.toString() ?? "null")
```

Comment:
Can we simply remove `.toString()` since `CategoricalChoiceType.value` is a string field.

```suggestion
    .map((c) => c?.value ?? "null")
```

Reply:
Apologies for missing this detail, thank you for the catch.